apps/mobile/src/components/headers/home-header.tsx

```diff
@@ -1,7 +1,21 @@
+import { NetworkBadge } from '@/features/settings/network-badge';
+
+import { Box } from '@leather.io/ui/native';
+
 import { HeaderLeatherLogo } from './components/header-leather-logo';
 import { HeaderOptions } from './components/header-options';
 import { HeaderLayout } from './header.layout';
 
 export function HomeHeader() {
-  return <HeaderLayout leftElement={<HeaderLeatherLogo />} rightElement={<HeaderOptions />} />;
+  return (
+    <HeaderLayout
+      leftElement={
+        <Box flexDirection="row" alignItems="center">
+          <HeaderLeatherLogo />
+          <NetworkBadge px="0" ml="-1" />
```

Comment:
I added this working off [this design](https://www.figma.com/design/FlmF7ropcijOb7iVSDqNn0/%F0%9F%9F%AA-App-Explorations-2024-H2?node-id=10194-400055&node-type=instance&m=dev).

In there the logo has less padding but I don't want to risk breaking something so giving this negative margin to move the badge into place:
![Screenshot 2024-12-09 at 06 53 59](https://github.com/user-attachments/assets/2b003445-3118-4609-842c-d4dd45cb3b73)